pycqed/instrument_drivers/physical_instruments/Fridge_monitor.py

```diff
@@ -127,19 +189,19 @@ def _update_monitor(self):
                 self.last_mon_update(upd_time.strftime('%Y-%m-%d %H:%M:%S'))
                 # Extract temperature names with temperature from source code
                 temperaturegroups = re.findall(
-                    r'<br>(T_[\w_]+(?: \(P\))?) = ([\S\.]+)', str(source))
+                    r'<br>(T_[\w_.]+(?: \(P\))?) = ([\S\.]+)', source)
 
                 self.temp_dict = {elem[0]: float(
                     elem[1]) for elem in temperaturegroups}
                 pressuregroups = re.findall(
-                    r'<br>(P_[\w_]+(?: \(P\))?) = ([\S\.]+)', str(source))
+                    r'<br>(P_[\w_.]+(?: \(P\))?) = ([\S\.]+)', source)
                 self.press_dict = {elem[0]: float(
                     elem[1]) for elem in pressuregroups}
 
-            except Exception as e:
-                logging.warning(e)
+            # except Exception as e:
```

Comment:
See comment above, if we decide against try/except, please remove it otherwise reimplement the try/except.